syntax = "proto3";

package addressbook;

import "person.proto";

service AddressBook {
  rpc AddPerson(Person) returns (NoContent) {}
  rpc ListPerson(NoArgs) returns (stream Person) {}
}

message NoContent {}

message NoArgs {}
